src/Bonobo.jl

```diff
@@ -47,6 +48,10 @@ mutable struct DefaultNode <: AbstractNode
     std :: BnBNodeInfo
 end
 
+function (::Type{AN})(args...) where AN
+    AN(BnBNodeInfo(), args...)
+end
```

Comment:
```suggestion
function (::Type{AN})(args...) where {AN <: AbstractNode}
    AN(BnBNodeInfo(), args...)
end
```
? Otherwise this is a type piracy for all constructors no?

Reply:
Oh yes definitely. Good catch